ownCloud/Settings/Passcode/AppLockManager.swift

```diff
@@ -146,6 +151,11 @@ class AppLockManager: NSObject {
 			} else {
 				passcodeViewController?.screenBlurringEnabled = forceShow
 			}
+
+            // Show biometrical
+            if !forceShow {
+                authenticateWithBiometrical()
+            }
```

Comment:
In general, please make sure to use tabs in this project, so it matches the rest of the code. Right now it looks like this on GitHub and in Xcode when using a tab-width other than 4:

<img width="698" alt="bildschirmfoto 2018-06-11 um 22 29 15" src="https://user-images.githubusercontent.com/639669/41255457-e7cd1062-6dc6-11e8-81a9-d607aeeeed3b.png">

 (no worries, if you really prefer spaces, you can change tab/space setting on a per-Xcode-project and per-file level, too)

Reply:
No sorry, I prefer tabs. I do not know why that comment have spaces instead tabs. I will change it